File: app/src/main/java/com/example/lpiem/pokecardapp/data/manager/api/GithubService.java
package com.example.lpiem.pokecardapp.data.manager.api;

import com.example.lpiem.pokecardapp.data.model.Github;

import java.util.List;

import retrofit2.http.GET;
import retrofit2.http.Path;
import rx.Observable;

public interface GithubService {
    @GET("users/{username}")
    Observable<Github> getGitHubUser(@Path("username") String userName);
    @GET("users")
    Observable<List<Github>> getGitHubUsers();
}

File: app/src/main/java/com/example/lpiem/pokecardapp/data/manager/api/GithubService.kt
package com.example.lpiem.pokecardapp.data.manager.api

import com.example.lpiem.pokecardapp.data.model.Github

import retrofit2.http.GET
import retrofit2.http.Path
import rx.Observable

interface GithubService {
    @get:GET("users")
    val gitHubUsers: Observable<List<Github>>

    @GET("users/{username}")
    fun getGitHubUser(@Path("username") userName: String): Observable<Github>
}
